modules/core/package.json

```diff
@@ -71,25 +71,11 @@
     ]
   },
   "deprecatedScripts": {
-    "build-script": "webpack",
```

Comment:
Just a heads up, this is the `deprecatedScripts` section you are modifying, so changes here won't have any effect. I created it recently when I was restructuring the repo just in case I missed something. Maybe just remove the entire thing to avoid confusion?